conf/igenomes.config

```diff
@@ -47,5 +47,24 @@ params {
       snpeffDb               = "GRCh38.86"
       vepCacheVersion        = "95"
     }
+    'GRCm38' {
+      acLoci                 = "${params.igenomes_base}/Mus_musculus/Annotation/ASCAT/fake.loci"
```

Comment:
I think we can remove fake files in configs, and directly not allow people to use ASCAT if no files are available.